common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -434,33 +473,36 @@
             1
         ],
         "import/no-named-as-default-member": [
-            1
+            "off"
+        ],
+        "import/no-nodejs-modules": [
+            "warn"
         ],
         "import/no-unassigned-import": [
             "error"
         ],
         "import/no-unresolved": [
-            2,
+            "error",
             {
                 "caseSensitive": true,
                 "caseSensitiveStrict": false
             }
         ],
         "import/no-unused-modules": [
-            "error"
+            "warn",
+            {
+                "missingExports": true
+            }
         ],
         "import/order": [
             "error"
         ],
-        "keyword-spacing": [
+        "indent": [
```

Comment:
Somewhere there needs to be an indent rule that errors

Reply:
I can't find one that works without heavily reformatting every file (prettier) or reliably (https://github.com/typescript-eslint/typescript-eslint/issues/1824: "The rule is broken and telling us different ways in which it is broken doesn't change the fact that it is broken.")

Another reason to prioritize tsfmt.